Add TPS730 LDO voltage regulator

matyas1995 opened the pull request:
Added TPS730285DBVT 200-mA, low-dropout voltage regulator with enable.
The new part is an alias of the TPS73625DBV 400-mA, low-dropout voltage regulator with identical footprint.
Datasheet:
https://www.ti.com/lit/gpn/TPS730

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

cpresser commented:
Hi and thanks for you contribution.

- [ ] The `T` suffix should not be part of the name (S2.2)
- [ ] Could you please also add the 1.8V variant? (not mandatory!)
- [x] The change to the symbol-outline and pin-position-offset are good. Please mention them in your commit message next time.

matyas1995 commented:
Hi,
I removed the T suffix from the name, added the 1,8V variant and found a link to the datasheet PDF. The link was just my error as I did not realize that TI only uses symlinks on their webpage.

cpresser commented on Regulator_Linear.dcm:
```diff
@@ -6972,6 +6972,24 @@ K linear low dropout Regulator fixed
+$CMP TPS730285DBVT
+D Low Noise, High PSRR, 200mA Low Drop 2.85V Regulator, SOT-23-5
+K Fixed LDO 200mA
+F https://www.ti.com/lit/gpn/TPS730
```
this should also get the direct .pdf link.

matyas1995 replied:
Oh, my bad. Similar to the https://github.com/KiCad/kicad-symbols/pull/2515 pull request, I first created a standalone symbol and later realized that I could also create it as an alias of an already existing compatible symbol. I will delete this duplicate.

cpresser commented on Regulator_Linear.lib:
```diff
@@ -4507,6 +4507,26 @@ X VOUT 5 300 100 100 L 50 50 1 1 w
 ENDDRAW
 ENDDEF
 #
+# TPS730285DBVT
```
I still see a `T` here.

matyas1995 replied:
Oh, my bad. Similar to the https://github.com/KiCad/kicad-symbols/pull/2515 pull request, I first created a standalone symbol and later realized that I could also create it as an alias of an already existing compatible symbol. I will delete this duplicate.
